README.md

```diff
@@ -56,20 +56,20 @@ for any matching payments and generated keys/addresses are saved to the local wa
 
 _**How are stealth and Enigma transactions detected/received?**_
 
-All incoming transactions are scanned. Stealth transactions are scanned for first (using the default ephemeral pubkey contained in a random OP_RETURN
```

Comment:
We _are_ really scanning the data _for_ stealth & Enigma transactions, as these are different from regular ones. Other than this, thank you for taking time to correct everything!